### PR TITLE
Move more logic from the distributive role into packages

### PR DESCRIPTION
**Pull request by siddharthist**
- [x] Installs cleanly on a fresh build of most recent master branch
- [ ] Upgrades cleanly from the most recent release
- [x] Updates documentation relevant to the changes

This should upgrade cleanly, but I haven't tested it. 

See https://github.com/asteris-llc/mantl-packaging/pull/81 for some more details. This essentially just moves the Ansible role logic into `install` scripts in an RPM.

### CODE REVIEW

File: roles/distributive/tasks/main.yml [hunk 1]
```diff
@@ -50,24 +10,5 @@
...
-- name: configure consul health check
-  become: yes
-  template:
-    src: consul-healthcheck.json.j2
-    dest: /etc/consul/distributive-{{ current_role }}-check.json
-  tags:
-    - bootstrap
-    - consul
-    - distributive
+    - "https://bintray.com/artifact/download/siddharthist/mantl-testing/distributive-{{ current_role }}-consul-config-{{ consul_config_versions[current_role] }}.x86_64.rpm"
```
Comment (siddharthist) [inline roles/distributive/tasks/main.yml]: This will have to be changed when https://github.com/asteris-llc/mantl-packaging/pull/81 is in.

**Comment by sehqlr** [comment]
Is this ready for testing? If so, any extra steps?

**Comment by siddharthist** [comment]
@sehqlr Should be ready for testing.

**Comment by siddharthist** [comment]
Closing due to lack of interest

**Pull request closed without merging**